portfolio/src/main/webapp/index.html

```diff
@@ -1,35 +1,57 @@
-      <img src="image/beau.jpg"/>
-      <p style="text-align:center">This is Beau, he loves water for some reason.</p>
-      <img src="image/pink1.jpg"/>
-      <p style="text-align:center">A nice picture of some Japanese cherry Trees I took.</p>
-      <img src="image/courthouse.jpg"/>
-      <p style="text-align:center">The downtown courthouse in my home town denton</p>
-      <img src="image/plants.jpg"/>
-      <p style="text-align:center">Some asthetically pleasing lants</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random community Quote:</p>
-      <button onclick="addRandomGreeting()">Quote</button>
-      <div id="greeting-container"></div>
-      <h3>Other Links.</h3>
-      <a href="https://www.linkedin.com/in/peter-emmanuel-abah-6b02a07a/"><img src="image/linkedin.png" style="width:150px;height:150px;"></a>
-      <a href="https://github.com/eoa0026"><img src="image/github.png" style="width:150px;height:90px;"></a>
-      
+        <h1>Peter's Portfolio</h1>
+        <p>This is my portfolio. A page to get a breif description of me</p>
+        <img src="image/profile.PNG" />
+        <h2>Summary</h2>
+        <p>Hello, I am peter, I am a Junior at Cal State Sacramento and I have been coding for about a year now, I am
+            from Texas but born in Nigeria and an international student and I really enjoy building software.</p>
+        <p>I really enjoy coding particularly in java and python, I really like the idea of being able to bring an idea
+            to real life through writing code. love photography and I do some of that in my free time. I have included
+            some links to my LinkedIn and GitHub profiles and some images that tell you a bit more about me</p>
+        <img src="image/beau.jpg" />
+        <p style="text-align:center">This is Beau, he loves water for some reason.</p>
+        <img src="image/pink1.jpg" />
+        <p style="text-align:center">A nice picture of some Japanese cherry Trees I took.</p>
+        <img src="image/courthouse.jpg" />
+        <p style="text-align:center">The downtown courthouse in my home town denton</p>
+        <img src="image/plants.jpg" />
+        <p style="text-align:center">Some asthetically pleasing Plants</p>
+        <p>Click this button to display a fun Sitcom Quote:</p>
+        <button onclick="getString()">Sitcom Quote</button>
+        <div id="hello-container"></div>
+        <p>Click here to get a random community Quote:</p>
+        <button onclick="addRandomGreeting()">Quote</button>
+        <div id="greeting-container"></div>
+        <h3>Contact Me:</h3>
+        <form action="/form-handler" method="POST">
+            <p>Name:</p>
+            <textarea name="text-input"></textarea>
```

Comment:
nit: Best practice is to name fields as something they represent, like `<textarea name="user-name"></textarea>`, `<textarea name="user-email"></textarea>`, etc.